client/css/style.css

```diff
@@ -2273,21 +2273,44 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 		height: 100%;
 		position: absolute;
 		left: -220px;
-		z-index: 1;
-		transition: transform 160ms;
+		z-index: 2;
+		transition: transform 160ms, box-shadow 0.2s;
```

Comment:
My understanding of transitioning `box-shadow` is that it could affect performance negatively (or look terrible on slow devices) and more common way is animating `opacity`.